### DISCUSSION TITLE
Hockey KeyError 124292?

### DISCUSSION THREAD

**Discussion by jwinik** [cwendt94/espn-api#235] [ESPN API] [answered]
Hi everyone- I successfully installed this package, but when I use `league = League(league_id = league_id, year = year, espn_s2 = espn_s2, swid = swid)`, I get `KeyError: 124292`.

Any ideas what is happening or how to troubleshoot? I tried using my username and password as arguments on top of what I posted above, which also did not work.

**Comment by cwendt94** [comment, marked as answer]
I just tried initializing on my league as well and got the same error. I think its because they added Seattle's new team Kraken and it is not defined. I will check it out, thanks for reporting this!

**Reply by jwinik** [reply]
That makes sense. Thanks for the quick response!

**Reply by cwendt94** [reply]
This has been fixed in package v0.18.1
